generator/integration_tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Updating the golden files
+
+We keep a set of "golden" files to ensure the generator has not changed its
+output in unexpected ways.  If you need to update these files, use this command:
```

Comment:
Question to ponder:  Now that we have actual generated interfaces to compare against, do the "golden" files have any value-add?

Reply:
I do not think they add much.  Let's talk about this on the next team meeting.

Reply:
FWIW, I would rather merge this as-is and discuss the removal later.  I am making more than one change to the generator and the current workflows are annoying.